rav1e_js/src/encoder_config.rs

```diff
@@ -270,3 +395,109 @@ impl EncoderConfig {
     self.clone()
   }
 }
+
+/// Supported Color Primaries
+///
+/// As defined by “Color primaries” section of ISO/IEC 23091-4/ITU-T H.273
+#[wasm_bindgen]
+pub enum ColorPrimaries {
```

Comment:
Not sure why are we re-defining here, 
We already have it in public API right, 
https://github.com/xiph/rav1e/blob/425c5ca9ccb08ff64c81f495e7fe109a891cae4f/src/api/color.rs#L48-L73

Reply:
https://docs.rs/rav1e/0.3.3/rav1e/color/enum.ColorPrimaries.html

Reply:
It is not possible to export these enums. See https://github.com/rustwasm/wasm-bindgen/issues/2273 for the problem.

Therefore I'm having a copy of them without annotated discriminants as a workaround until it is possible to export them.

Reply:
Please add a note about it.